classes/classroom.rb

```diff
@@ -0,0 +1,13 @@
+class Classroom
+  attr_accessor :label
+  attr_reader :students
+
+  def initialize(label)
+    @label = label
+    @students = []
+  end
+
+  def add_student(student)
+    @students << student unless @students.include?(student)
```

Comment:
- [ ] Kindly add a relation to classroom (classroom has many students) by adding this line `student.classroom = self`

Reply:
Oh thank you @omarsalem7 let me implement this